documentation/staging/content/userguide/platforms/environments.md

```diff
@@ -46,14 +50,34 @@ see [Operator prerequisites]({{< relref "/userguide/prerequisites/introduction.m
 
 ### Pricing and licensing
 
+The WebLogic Kubernetes Operator and Oracle Linux are open source and free;
+WebLogic Server requires licenses unless used in a single developer desktop development environment.
+In detail:
+
+#### WebLogic Kubernetes Operator
+
 The WebLogic Kubernetes Operator (the "operator") is open source and free,
 licensed under the Universal Permissive license (UPL), Version 1.0.
 
+#### WebLogic Server
+
 WebLogic Server is not open source.
 Licensing is required for each running WebLogic Server instance in Kubernetes,
 just as with any deployment of WebLogic Server.
 Licensing is free for a single developer desktop development environment.
 
+#### Oracle Linux and WebLogic Container Images
+
+Oracle Linux is under open source license and is completely free to download and use.
+- In addition, with WebLogic license and support,
+  customers have access to the latest WebLogic container images which bundle the latest slim Oracle Linux images.
+- Customer support for WebLogic container images is handled by the WebLogic support team.
+- Oracle Linux patches are delivered only with the newer WebLogic container images,
```

Comment:
Oracle Linux fixes including security fixes are delivered as new images.  For WebLogic to uptake these fixes new WebLogic images based on the new Oracle Linux images need to be created.

Reply:
Conversation continues via DM in slack.